src/Compilers/VisualBasic/Portable/Binding/Binder_Attributes.vb

```diff
@@ -787,7 +787,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                                         If cast.Keyword.Kind = SyntaxKind.CObjKeyword Then
                                             InternalSyntax.Parser.CheckFeatureAvailability(diagBag,
                                                                                            cast.Keyword.GetLocation(),
-                                                                                           DirectCast(cast.SyntaxTree, VisualBasicSyntaxTree).Options.LanguageVersion,
+                                                                                           DirectCast(cast.SyntaxTree, VisualBasicSyntaxTree).Options,
```

Comment:
Note: This changes allows use to check the presence of command line feature flags,